### ISSUE TITLE
A11y_AzureDataStudioSQL_Projects_Publish Projects_ScreenReader: Voice Over announces control type of database Combo box as Button.

### ISSUE THREAD

**Issue by RohitJadhav-TCS** [microsoft/azuredatastudio#14523]
### Actual:
Voice Over announces control type of database Combo box as Button.

### Expected:
Voice Over should announce the control type for Database combo box as combo box.

**GitHubTags**: #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudioSQL-MAC-Feb2021;#SQL Azure Data Studio;#MAC;#FTP;#DesktopApp;#WCAG4.1.2;#VoiceOver;#Element:Combobox:#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio
Version: 1.26.1
Mac OS Version: Big Sur Version 11.2.2
Screen reader: Voice Over

### Repro Steps:

1. Launch the application-Azure Database Project and install extension SQL Database Projects.
2. Navigate to SQL Database Project Home Screen with heading "Projects".
3. Tab to connections select your server and connect to it.
4. Tab to Projects tab an right click on any created project.
5. Tab till "Publish Project" and hit enter
6. Start Voice Over and tab to database combo box and check the result.

### User Impact:
Screen reader user will get confused if Voice Over announces control type of database Combo box as Button.

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Comment by kisantia** [comment]
As decided in https://github.com/microsoft/azuredatastudio/issues/5679 and https://github.com/microsoft/azuredatastudio/issues/6847, this is the expected behavior on Mac. Can we close this?

**Comment by kisantia** [comment]
@RohitJadhav-TCS can we close this as expected behavior?

**Comment by kisantia** [comment]
closing as expected behavior on Mac, as previously approved in #5679 and #6847

@hannahqin fyi

**Comment by Swati1700** [comment]
Verified the bug in the environment
Application Name: Azure Data Studio
macOS: Ventura 13.2
Issue is fixed and does not reproduced. Please find the not repro attachment below. 
[Screen Recording (2) (1).webm](https://user-images.githubusercontent.com/86710888/220584461-fba78f7a-e2d6-4ecd-8ad2-3ad84ea4d49b.webm)
